internal/store/builder.go

```diff
@@ -107,18 +107,26 @@ func (b *Builder) WithMetrics(r prometheus.Registerer) {
 
 // WithEnabledResources sets the enabledResources property of a Builder.
 func (b *Builder) WithEnabledResources(r []string) error {
+	enabledResources := map[string]bool{}
```

Comment:
Can you remove this bugfix here (as it shouldn't be part of this change)?

We have another PR that tries to solve the same issue here: https://github.com/kubernetes/kube-state-metrics/pull/2502

Reply:
Sure, makes sense. I needed it for the e2e test but I just changed the validation